pytorch/inference/docker/2.2/py3/Dockerfile.graviton.cpu

```diff
@@ -0,0 +1,253 @@
+ && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+ && cd openmpi-$OPEN_MPI_VERSION \
+ && ./configure --prefix=/home/.openmpi \
+ && make all install \
+ && cd .. \
+ && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# The ENV variables declared below are changed in the previous section
+# Grouping these ENV variables in the first section causes
+# ompi_info to fail. This is only observed in CPU containers
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
+
+# Install CondaForge miniconda
+RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-aarch64.sh \
+ && chmod +x ~/mambaforge.sh \
+ && ~/mambaforge.sh -b -p /opt/conda \
+ && rm ~/mambaforge.sh \
+ && /opt/conda/bin/conda install -c conda-forge -y python=$PYTHON_VERSION \
+    ruamel_yaml \
+    cython \
+    ipython \
+    parso \
+    # Below 2 are included in miniconda base, but not mamba so need to install
+    conda-content-trust \
+    charset-normalizer \
+    requests \
+ && conda clean -ya
+
+# Install Common python packages
+RUN ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && pip install \
+    boto3 \
+    scipy \
+    opencv-python \
+    "pyOpenSSL>=24.0.0" \
+    "cryptography>=41.0.2" \
+    "ipython>=8.10.0,<9.0" \
+    "awscli<2" \
+    "urllib3>=1.26.18,<2"
+
+# Ensure PyTorch did not get installed from Conda or pip, prior to now
+RUN pip uninstall -y torch torchvision torchaudio torchdata model-archiver multi-model-server
+
+# install pytorch wheels
+RUN pip install --no-cache-dir -U \
+    ${TORCH_URL} \
+    ${TORCHVISION_URL} \
+    ${TORCHAUDIO_URL} \
+    ${TORCHTEXT_URL}
+
+# Install TorchServe pypi dependencies directly from their requirements.txt file
+# NOTE: This also brings in unnecessary cpu dependencies like nvgpu
+# -> Numpy is higer version as required by torch
+RUN pip install --no-cache-dir -U -r https://raw.githubusercontent.com/pytorch/serve/v${TORCHSERVE_VERSION}/requirements/common.txt \
+ && pip install --no-cache-dir -U \
+    "numpy>=1.22.2,<1.23" \
+    torchserve==${TORCHSERVE_VERSION} \
+    torch-model-archiver==${TORCHSERVE_VERSION}
+
+# add necessary certificate for aws sdk cpp download
+RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+
+# create user and folders
+RUN useradd -m model-server \
+ && mkdir -p /home/model-server/tmp /opt/ml/model \
+ && chown -R model-server /home/model-server /opt/ml/model
+
+# add TS entrypoint
+COPY torchserve-ec2-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY config.properties /home/model-server/config.properties
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.py
```

Comment:
Would probably require an update if we do update the config. Reach out to TS team.